templates/splash.html

```diff
@@ -265,6 +265,27 @@
       }
     });
   });
+
+  // Script for forcing browser to go full screen
+  function handleConfirmation() {
+    $("#permissions-modal").removeClass("is-active");
+    confirmationDismissed = true;
+    goFullScreen();
```

Comment:
Personally, I could see this behavior of my browser going full screen to be unexpected. I personally prefer to work in windowed mode during active development. Another situation is if I'm in a multi-monitor setup and I wanted to drag the window to another 

Maybe make function take a `fullscreen: boolean` argument and add a separate button below confirm? e.g. "Confirm and go full screen"

Reply:
Hmm.... ya, I see your point. I was really just trying to consolidate functions since a button press was already required. I've been hitting escape to get out of full-screen mode for dev work. I like your suggestion though.

Reply:
A full screen button has been implemented, marking as resolved.